### PR TITLE
Sub-basin: Add AOI-Level Source Table

### PR DESCRIPTION
**Pull request by arottersman**
## Overview

Displays the attenuated results for the AOI.

Connects #2724 

### Demo
screen shot 2018-04-16 at 12 45 31 pm

screen shot 2018-04-16 at 12 45 40 pm

## Testing Instructions

 * Pull and bundle.
 * Select a huc-10 with multi-year model
 * On the Water Quality tab, click "view attenuated results"
 * When the subbasin job has finished you should see the table picture above. You should be able to horizontally scroll it. The last three columns should be equivalent to the second to last three columns over the area column. The areas used and displayed in the area column should follow what's described by Anthony in the original issue.

### CODE REVIEW

File: src/mmw/js/src/modeling/gwlfe/subbasin/templates/sourcesTable.html [hunk 1]
```diff
@@ -1 +1,49 @@
...
+                <td class="strong text-right">{{ row.TotalP|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ (row.Sediment/row.Area)|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ (row.TotalN/row.Area)|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ (row.TotalP/row.Area)|round(2)|toLocaleString(2) }}</td>
+            </tr>
+        {% endfor %}
+    </tbody>
+    <tfoot>
+        <tr>
+            <th class="text-left">{{ summaryRow.Source }}</td>
```
Comment (arottersman) [inline src/mmw/js/src/modeling/gwlfe/subbasin/templates/sourcesTable.html]: Will update this tomorrow to not start with `<th>` and close with `</td>`. Doesn't seem to affect it for testing or anything though.

Comment (arottersman) [reply src/mmw/js/src/modeling/gwlfe/subbasin/templates/sourcesTable.html]: updated!

**Comment by rajadain** [comment]
The "Download This Data" button does not work.

**Comment by arottersman** [comment]
> The "Download This Data" button does not work.

It's not wired up to anything. I'll make a note in the commit

**Comment by arottersman** [comment]
Thanks! Fixup squashed; will merge when built. There's https://github.com/WikiWatershed/model-my-watershed/issues/2656 as a placeholder for wiring up all the download buttons.

**Pull request merged**